crates/bin/pd/src/main.rs

```diff
@@ -328,7 +328,7 @@ async fn main() -> anyhow::Result<()> {
                     let randomizer = OsRng.gen::<u32>();
                     let chain_id =
                         chain_id.unwrap_or_else(|| env!("PD_LATEST_TESTNET_NAME").to_string());
-                    format!("{}-{}", chain_id, hex::encode(randomizer.to_le_bytes()))
+                    format!("{}-{}", chain_id, randomizer)
```

Comment:
N.B. this will update preview/devnet chain ids to a format like `penumbra-testnet-deimos-8-722430144`, where the lattermost chunk is understood as a revision. Off the cuff, it might make more sense for the cli to default to rev 0 for devnets, but that's a bikeshedding conversation for another day. Thanks for fixing!